test/runScript-mock-pMap.spec.js

```diff
@@ -12,21 +12,27 @@ const packageJSON = {
 }
 
 describe('runScript', () => {
+  afterEach(() => {
+    pMapMock.mockClear()
+  })
+
   it('should respect concurrency', () => {
+    pMapMock.mockImplementation(() => Promise.resolve(true))
+
     const res = runScript(['test'], ['test1.js', 'test2.js'], packageJSON, {
       chunkSize: 1,
       subTaskConcurrency: 1
     })
     res[0].task()
-    expect(pMap.mock.calls.length).toEqual(1)
-    const pMapArgs = pMap.mock.calls[0]
+    expect(pMapMock.mock.calls.length).toEqual(1)
```

Comment:
`expect(pMapMock).toHaveBeenCalledWith(...)`?

http://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwitharg1-arg2-

Reply:
This isn't the only place(ex: [`runScript.js`](https://github.com/okonet/lint-staged/blob/master/test/runScript.spec.js#L51)). I'll prefer doing these changes in a different PR.